server/lib/house/house.destroy.js

```diff
@@ -20,6 +21,7 @@ async function destroy(selector) {
   }
 
   await house.destroy();
+  this.event.emit(EVENTS.HOUSE.DELETE);
```

Comment:
maybe we want to send the deleted house here?

```
this.event.emit(EVENTS.HOUSE.DELETED, house.get({ plain: true}));
```

Even if it's not useful in this PR, I think it make sense for future uses